code/LINDA/LINDA_fire.dm

```diff
@@ -126,6 +126,9 @@
 			var/radiated_temperature = location.air.temperature*FIRE_SPREAD_RADIOSITY_SCALE
 			for(var/direction in cardinal)
 				if(!(location.atmos_adjacent_turfs & direction))
+					var/turf/simulated/wall/W = get_step(src, direction)
+					if(istype(W))
+						W.adjacent_fire_act(W, radiated_temperature)
```

Comment:
TG also killed off all unsimulated turfs--everything is fully simulated minus like..space.